### ISSUE TITLE
To many bbox on frame

### ISSUE THREAD

**Issue by ViktorPavlovA** [cluangar/YOLOv5-RK3588-Python#2]
![Screenshot from 2024-05-23 14-02-20](https://github.com/cluangar/YOLOv5-RK3588-Python/assets/74448820/adc79c2f-5bdf-4d58-8142-a40ccb95b301)

Hello. Can you help me a bit? How you sort bbox on frame ?

**Comment by cluangar** [comment]
Please check torch, opencv version. I test on opencv 4.6.0.66, 4.7.0 and torch 1.13.1.
My system use on this
asttokens==2.2.1
backcall==0.2.0
certifi==2022.12.7
charset-normalizer==2.1.1
click==8.1.3
contourpy==1.0.6
cycler==0.11.0
decorator==5.1.1
dlib==19.24.0
executing==1.2.0
face-recognition==1.3.0
face-recognition-models==0.3.0
filelock==3.14.0
fonttools==4.38.0
fsspec==2024.5.0
gitdb==4.0.10
GitPython==3.1.29
idna==3.4
imageio==2.34.1
imutils==0.5.4
ipython==8.7.0
jedi==0.18.2
Jinja2==3.1.4
joblib==1.2.0
kiwisolver==1.4.4
lazy_loader==0.4
MarkupSafe==2.1.5
matplotlib==3.6.2
matplotlib-inline==0.1.6
mpmath==1.3.0
networkx==3.2.1
ninja==1.11.1.1
numpy==1.24.1
opencv-python==4.6.0.66
packaging==22.0
pandas==1.5.2
parso==0.8.3
pexpect==4.8.0
pickleshare==0.7.5
Pillow==9.3.0
prompt-toolkit==3.0.36
psutil==5.9.4
ptyprocess==0.7.0
pure-eval==0.2.2
pycairo==1.23.0
pyclipper==1.3.0.post5
Pygments==2.13.0
PyGObject==3.42.2
pyparsing==3.0.9
python-bidi==0.4.2
python-dateutil==2.8.2
pytz==2022.7
PyYAML==6.0
requests==2.28.1
rknn-toolkit-lite2 @ file:///home/orangepi/rknn_toolkit_lite2-1.4.0-cp39-cp39-linux_aarch64.whl
ruamel.yaml==0.17.21
ruamel.yaml.clib==0.2.7
scikit-image==0.22.0
scikit-learn==1.2.0
scipy==1.9.3
seaborn==0.12.1
shapely==2.0.4
six==1.16.0
sklearn==0.0.post1
smmap==5.0.0
stack-data==0.6.2
sympy==1.12
threadpoolctl==3.1.0
tifffile==2024.5.22
torch==1.13.1
torchvision==0.14.1
tqdm==4.64.1
traitlets==5.8.0
typing_extensions==4.11.0
urllib3==1.26.13
wcwidth==0.2.5
